native/cocos/application/BaseGame.cpp

```diff
@@ -50,16 +50,16 @@ int BaseGame::init() {
     });
 
 #endif
-
-    if (_debuggerInfo.enabled) {
-        setDebugIpAndPort(_debuggerInfo.address, _debuggerInfo.port, _debuggerInfo.pauseOnStart);
-    }
-
+    
     int ret = cc::CocosApplication::init();
     if (ret != 0) {
         return ret;
     }
 
+    if (_debuggerInfo.enabled) {
+        setDebugIpAndPort(_debuggerInfo.address, _debuggerInfo.port, _debuggerInfo.pauseOnStart);
```

Comment:
Why change the order of setDebugIpAndPort after CocosApplication::init? 
This change will make the JS debugger disabled all the time even in debug mode.
<img width="753" alt="Screen Shot 2022-06-03 at 23 16 52" src="https://user-images.githubusercontent.com/493372/171882903-8ffd5ad0-a8b3-453b-bd78-84bbf7ce8979.png">

Reply:
As it depends on ScriptEngine, so should execute the codes after ScriptEngine is created.